purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/offlineentitlements/OfflineEntitlementsFreshInstallIntegrationTest.kt

```diff
@@ -21,6 +52,7 @@ class OfflineEntitlementsFreshInstallIntegrationTest : BaseOfflineEntitlementsIn
     fun setUp() {
         ensureBlockFinishes { latch ->
             setUpTest {
+                mockBillingAbstract.mockQueryProductDetails()
```

Comment:
Now, when we getCustomerInfo, if there are pending purchases (like in these tests with initial purchases), we will try to get information about the product and post it, so this was required.